1-Computer-Vision/Exercises/1-3-2-Stochastic-Gradient-Descent/2022-08-29-Stochastic-Gradient-Descent.ipynb

```diff
@@ -46,7 +46,7 @@
   },
```

Comment:
<p>Re:</p><blockquote>Looks like mean validation accuracy dropped slightly in your experiment run.</blockquote><p> </p><p>Looks like with the Apple Silicon architecture you have "enough" cache to fit the batch in memory (running M2 Max, correct?). Not sure your training batch size would be automatically reduced because of this...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jonathanloganmoran/ND0013-Self-Driving-Car-Engineer/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Rerun everything using GPU now. It is weird that the training speed of Nvidia 3090 is not as fast as Apple M2. It is aware that there is about 20% performance decay using docker. The good news is that the final performance of the trained model seems fine now.</p>